index.js

```diff
@@ -35,6 +36,23 @@ function rehypeReact(options) {
   function h(name, props, children) {
     var component = name
 
+    // Currently, a warning is triggered by react for *any* white space in
+    // tables.
+    // So we remove the pretty lines for now.
+    // See: <https://github.com/facebook/react/pull/7081>.
+    // See: <https://github.com/facebook/react/pull/7515>.
+    // See: <https://github.com/remarkjs/remark-react/issues/64>.
+    if (children && tableElements.indexOf(name) !== -1) {
+        children = children.filter(function (child) {
+            const notNewline = child !== '\n' && child !== '\r';
+            let notEmpty = true;
+            if (typeof child === 'string') {
+                notEmpty = child.trim().length > 0;
+            }
+            return notNewline && notEmpty;
```

Comment:
Can you instead use the functionality that is used here: https://github.com/remarkjs/remark-react/blob/d4311dd0f91813b08d0ca232f112b420e1ac0979/index.js#L47-L49. Removing *just* `'\n'`?

Reply:
why? I've seen tables created that have space nodes that look like they are also for formatting and do not relate to the content put into the table

Reply:
ex: https://github.com/rehypejs/rehype-react/issues/28#issuecomment-862388914

Reply:
I only see `\n` in those cases (and applying this fix to several projects has not been a problem). Could you make that reproducible?

Reply:
apologies it is just \n, the codesandbox didn't render it escaped so it was "\n" in reality but looked empty. I will adjust this